core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5267,7 +5273,15 @@ private static void computeVersion(ServletContext context) {
     private static final Logger LOGGER = Logger.getLogger(Jenkins.class.getName());
 
     public static final PermissionGroup PERMISSIONS = Permission.HUDSON_PERMISSIONS;
+    /**
+     * Grants ability to configure any and all aspects of the Jenkins instance
+     */
     public static final Permission ADMINISTER = Permission.HUDSON_ADMINISTER;
+    /**
+     * Allows non-privilege escalating configuration permission for a Jenkins instance.  Actions which could result
+     * in a privilege  escalation (such as RUN_SCRIPTS) require explicit ADMINISTER permission
```

Comment:
If `RUN_SCRIPTS` is now deprecated, why are you referring to it?

Reply:
the deprecation is being handled in a separate PR, hence we act as if it is still around here...

Reply:
For the record, the PR on RUN_SCRIPT is there https://github.com/jenkinsci/jenkins/pull/4365